.github/ISSUE_TEMPLATE/content-request-.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Content request '
+about: Suggest an idea for additional content to add to this page
+title: "[Addition] TITLE (a short summary of your request)"
+labels: feature
+assignees: annNau
+
+---
```

Comment:
Suggestions for additional pages, possibly including submission of articles (pending decision on a fixed procedure for the latter).